src/builder-git.c

```diff
@@ -582,7 +582,9 @@ builder_git_mirror_repo (const char     *repo_location,
           if (!git (mirror_dir, NULL, 0, error,
                     "lfs", "fetch", "--all", NULL))
             {
-              git (mirror_dir, NULL, 0, error,
+              g_autoptr(GError) log_error = NULL;
```

Comment:
Ahh, this really should be squashed into the previous commit; but since you don't care about the error here you can just pass NULL to the `git lfs logs` call.